src/vorta/tray_menu.py

```diff
@@ -73,10 +73,19 @@ def build_menu(self):
         exit_action = menu.addAction(self.tr('Quit'))
         exit_action.triggered.connect(self.app.quit)
 
+    def is_taskbar_dark(self):
+        app = QApplication.instance()
```

Comment:
`self.app` should already have the QApplication instance.

Reply:
Changed it! It works now, earlier it gave error before PyQt6.